exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillConvertletTable.java

```diff
@@ -36,19 +37,36 @@
   public static HashMap<SqlOperator, SqlRexConvertlet> map = new HashMap<>();
 
   public static SqlRexConvertletTable INSTANCE = new DrillConvertletTable();
-  private static SqlRexConvertlet sqrtConvertlet = new SqlRexConvertlet() {
-    public RexNode convertCall(SqlRexContext cx, SqlCall call) {
-      RexNode operand = cx.convertExpression(call.operand(0));
-      return cx.getRexBuilder().makeCall(SqlStdOperatorTable.SQRT, operand);
+
+  private static final SqlRexConvertlet SQRT_CONVERTLET = (cx, call) -> {
+    RexNode operand = cx.convertExpression(call.operand(0));
+    return cx.getRexBuilder().makeCall(SqlStdOperatorTable.SQRT, operand);
+  };
+
+  // Rewrites COALESCE function into CASE WHEN IS NOT NULL operand1 THEN operand1...
+  private static final SqlRexConvertlet COALESCE_CONVERTLET = (cx, call) -> {
+    if (call.operandCount() == 1) {
```

Comment:
can you create a local variable something like argCount for call.operandCount() and use it.